projects/plugins/crm/tests/php/automation/mocks/mock-class-contact-created-trigger.php

```diff
@@ -33,7 +33,7 @@ public static function get_description(): ?string {
 	 * @return string
 	 */
 	public static function get_category(): ?string {
-		return __( 'contact', 'zero-bs-crm' );
+		return 'contact';
```

Comment:
```suggestion
		return __( 'Contact', 'zero-bs-crm' );
```

Reply:
It's tests context. We don't need to translate it there.